<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>117</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>117</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>117</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>99</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>99</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>142</red>
        <green>99</green>
        <blue>227</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>MPlayer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="buttons.qrc">
    <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>0.700000000000000</double>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="btn_add">
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/plus.png</normaloff>:/buttons/plus.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="btn_stop">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/stop.png</normaloff>:/buttons/stop.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QToolButton" name="btn_next">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/skip-next.png</normaloff>:/buttons/skip-next.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="btn_previous">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/skip-previous.png</normaloff>:/buttons/skip-previous.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="btn_play">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/play.png</normaloff>:/buttons/play.png</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btn_pause">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="buttons.qrc">
       <normaloff>:/buttons/pause.png</normaloff>:/buttons/pause.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="8">
    <widget class="QTableView" name="playlistView"/>
   </item>
   <item row="0" column="7">
    <widget class="QLabel" name="currentTrack">
     <property name="font">
      <font>
       <family>Verdana</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="buttons.qrc"/>
 </resources>
 <connections/>
</ui>
